en/option/component/axis-common.md

```diff
@@ -389,8 +389,7 @@ min: function (value) {
 }
 ```
 
-
-#${prefix} max(number|string) = null
+#${prefix} max(number|string|function) = null

```

Comment:
`Function` instead of `function` because `number` and `string` are primary types while `Function` is Object.